prism/src/main/java/org/apache/falcon/resource/AbstractInstanceManager.java

```diff
@@ -173,9 +174,16 @@ public InstancesResult getStatus(String type, String entity, String startStr, St
 
             // LifeCycle lifeCycleObject = EntityUtil.getLifeCycle(lifeCycle);
             AbstractWorkflowEngine wfEngine = getWorkflowEngine(entityObject);
-            return getInstanceResultSubset(wfEngine.getStatus(entityObject,
-                            startAndEndDate.first, startAndEndDate.second, lifeCycles, allAttempts),
+            InstancesResult instancesResult = getInstanceResultSubset(wfEngine.getStatus(entityObject,
```

Comment:
Yes it used to return after applying numResults but I have move that code to this method from line number 310 to 184.Now it will give us i11-i15.

I have also pasted the same in the testing.